src/Vizkit3dWorld.cpp

```diff
@@ -155,6 +164,14 @@ vizkit3d::RobotVisualization* Vizkit3dWorld::robotVizFromSdfModel(sdf::ElementPt
     //vizkit3d plugin with model defined in the sdf model
     vizkit3d::RobotVisualization* robotViz = new vizkit3d::RobotVisualization();
 
+    for (int i=0; i < excludeFromSdf.size(); ++i){
```

Comment:
Hi....I still didn't figure out why the pop up appears. I'll go deeper on that tomorrow, but it happen not specifically with flat_fish, but with the sdf that we want to exclude (see the mega_structure case)
![erro2](https://cloud.githubusercontent.com/assets/8384920/14661976/7c8a4b46-0688-11e6-81dd-e804ed08c6e0.png)

However after I click the "ok" button on the pop up the camera starts to run normally. It ignores the sdf files now (I realized that was doing some stuffs wrong yesterday).

So, now I agree with just ignore the sdf instead of removing the visual and collision elements
But still need to figure out how to prevent the pop up messages.